lms/djangoapps/course_home_api/progress/v1/serializers.py

```diff
@@ -27,13 +27,24 @@ class SubsectionScoresSerializer(serializers.Serializer):
     num_points_earned = serializers.FloatField(source='graded_total.earned')
     num_points_possible = serializers.FloatField(source='graded_total.possible')
     percent_graded = serializers.FloatField()
+    problem_scores = serializers.SerializerMethodField()
```

Comment:
This field is not "fancy" in that it is just a list of simple serialization - there might be an existing serializer that handles it, like a DictField(many=True) or something? But this is fine too.

Reply:
If I try to use anything else here I get `TypeError: Object of type BlockUsageLocator is not JSON serializable`.